mlprojects/sklearn-diabetes/MLproject

```diff
@@ -0,0 +1,9 @@
+name: tutorial
```

Comment:
consider updating this to something more meaningful/standardizing the MLproject names across the repo